packages/connect/src/types/firmware.ts

```diff
@@ -17,6 +17,7 @@ export type FirmwareRelease = {
     fingerprint: string;
     changelog: string | string[];
     changelog_bitcoinonly?: string | string[]; // Added later, may not be there for older releases
+    firmware_revision?: string;
```

Comment:
this is confusing,
i dont see `firmware_revision` field in either online (https://data.trezor.io/firmware/t1b1/releases.json) or offline (data dir) releases jsons.


does it mean it will fail here or i missed some deeper logic behind it?

```
if (onlineRelease?.firmware_revision === undefined) {
    return failFirmwareRevisionCheck('firmware-version-unknown');
}
```

Reply:
It will be there on data.trezor.io. @MiroslavProchazka shall finish his PR https://github.com/trezor/data/pull/88 :grin:

Reply:
If the FW release is not found locally AND its not found online it will show ghosts. 

So it is absolutely **crucial** to not forget to add `firmware_revision` to the data.trezor.io **before** we release new FW version.

Reply:
if its crucial then it should be covered by some pre-release e2e test

Reply:
also this PR should get the update of releases.json before merging, right?

Reply:
@mroz22 already done https://github.com/trezor/trezor-suite/pull/13502